dts/arm/microchip/mec172x/mec172xnsz-pinctrl.dtsi

```diff
@@ -621,24 +617,28 @@
 		pinmux = < MCHP_XEC_PINMUX(0126, MCHP_AF1) >;
 	};
 
-	gpspi_cs_n_gpio116: gpspi_cs_n_gpio116 {
-		pinmux = < MCHP_XEC_PINMUX(0116, MCHP_AF1) >;
+	gpspi_cs_n_gpio024: gpspi_cs_n_gpio024 {
+		pinmux = < MCHP_XEC_PINMUX(024, MCHP_AF1) >;
```

Comment:
We are reviewing gpspi nomenclature used in the specification and confirm if this change is required.